### PR TITLE
PR: Content rework for HMO

### PR DESCRIPTION
**Pull request by lizziebruce**
Content fix for #313 - clear explanation of an HMO and next steps for integrated and non-integrated user journey.

### CODE REVIEW

**Review by EUzkuraityte — APPROVED** [review]

File: app/councils.json [hunk 1]
```diff
@@ -121,6 +122,7 @@
     "parkingBoundary": "Ashbrooke and Thornholme",
     "boundaryLink": "http://www.sunderland.gov.uk/CHttpHandler.ashx?id=18284&p=0&fsize=883kb&ftype=Ashbrooke%20and%20Thornholme%20CPMS.PDF",
     "boundaryPage": "parking-boundary-map",
+    "hmo": "phone",
```
Comment (suryanaga) [inline app/councils.json]: @lizziebruce What does this do? Doesn't look like you've used it in the HTML to me

**Comment by lizziebruce** [comment]
@sanjaypoyzer for some councils if you live in an HMO you have to complete your application by phone, there isn't online integration

**Comment by suryanaga** [comment]
@lizziebruce So it looks maybe you haven't committed the `hmo-info.html` file?

**Comment by lizziebruce** [comment]
@sanjaypoyzer that's why I need help with this stuff... I've only been using Atom for a few weeks so I'm gonna to get a few things wrong when we're not pair working

**Pull request closed without merging**